.github/workflows/manual-deploy.yml

```diff
@@ -100,10 +101,18 @@ jobs:
       cluster: ${{ inputs.environment }}
       service-name: "boxel-pg-migration-${{ inputs.environment }}"
       image: ${{ needs.build-pg-migration.outputs.image }}
+      wait-for-service-stability: false
+
+  post-migrate-db:
+    name: Wait for db-migration
+    needs: [migrate-db]
+    runs-on: ubuntu-latest
+    steps:
+      - run: sleep 240
```

Comment:
we still need to wait for the ECS task to be provisioned and start running. (the old action  version actually just waited for the timeout of 5 minutes--i think we can do a little better than that)